### PR TITLE
rust-openid-token-plugin integration

### PR DESCRIPTION
**Pull request by Ross-ForAllSecure**

### CODE REVIEW

File: .github/workflows/mayhem.yml [hunk 1]
```diff
@@ -108,8 +129,8 @@ jobs:
           mapi-token: ${{ secrets.MAPI_TOKEN }}
           api-url: http://localhost:8080/admin/realms
           api-spec: .github/api/openapi_16.0.yml
-          target: mayhemheroes/keycloak-quarkus
-          duration: 1min
+          target: rossimo/keycloak-quarkus
```
Comment (vicero) [inline .github/workflows/mayhem.yml:132]: Did you mean to change this?

File: .github/workflows/mayhem.yml [hunk 2]
```diff
@@ -79,16 +84,32 @@ jobs:
...
-          token=$( curl -X POST \
-            -H 'Content-Type: application/x-www-form-urlencoded' \
-            -d 'username=admin&password=admin&client_id=admin-cli&grant_type=password' \
-            http://localhost:8080/realms/master/protocol/openid-connect/token | jq -j ".access_token")
-          echo "AUTH_TOKEN=$token" >> "$GITHUB_ENV"
+          sudo apt update
+          sudo apt install -y protobuf-compiler libprotobuf-dev
+
+          cd $GITHUB_WORKSPACE/../
+          git clone https://github.com/ForAllSecure/mapi-examples
```
Comment (vicero) [inline .github/workflows/mayhem.yml:93]: Optionally I think you could use the [checkout action](https://github.com/actions/checkout) instead to simplify this step

**Review by vicero — CHANGES_REQUESTED** [review]

File: .github/workflows/mayhem.yml (hunk 2 above)
Comment (Ross-ForAllSecure) [reply .github/workflows/mayhem.yml:93]: Yeah, I tried that, but then it checks it out into the same directory as keycloak.  Couldn't get that path to work.

File: .github/workflows/mayhem.yml (hunk 1 above)
Comment (Ross-ForAllSecure) [reply .github/workflows/mayhem.yml:132]: Good catch. Thanks!

**Pull request merged**